lib/index.js

```diff
@@ -62,12 +74,11 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       store: ['validate', (results, callback) => {
         // generate new random ID for inbox
         const inbox = results.validate;
-        inbox.id = routes.inboxes + '/' + uuidV4();
-        const { owner: owner=req.user.identity.id } = inbox;
+        inbox.id = api.createInboxId();
+        const {owner: owner = req.user.identity.id} = inbox;
         delete inbox.owner;
-        brLdnInbox.inboxes.add(
-          req.user.identity, inbox, {owner: owner},
-          err => callback(err, inbox.id));
+        brLdnInbox.inboxes.add(req.user.identity, inbox, {owner: owner}, err =>
+          callback(err, inbox.id));
```

Comment:
I thought moving the whole `err => callback(err, inbox.id)` line to its own line was clearer here, since it all fit on one line. Do you disagree?